paddle/infrt/backends/host/pten_allocator.h

```diff
@@ -0,0 +1,33 @@
+/* Copyright (c) 2022 PaddlePaddle Authors. All Rights Reserved.
```

Comment:
infrt/backends/pten/host
infrt/backends/pten/gpu
infrt/backends/others/host

or

infrt/backends/host/pten
infrt/backends/host/others

Reply:
> infrt/backends/pten/host infrt/backends/pten/gpu infrt/backends/others/host
> 
> or
> 
> infrt/backends/host/pten infrt/backends/host/others

ok, I will fix in next commit